pandas/tests/util/test_assert_almost_equal.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
 
```

Comment:
Probably worth deleting leading whitespace (applicable in a few modules)

Reply:
Ok, slightly confusing comment coming:

I saw that just after pushing, and was working with ``-# coding=utf-8`` at the time, so I changed this PR to deal with that (including dealing with whitespace).